### DISCUSSION TITLE
Use version numbering in requirements.txt

### DISCUSSION THREAD

**Discussion by erlenddalen** [synaptiai/uim-protocol#4] [Ideas]
When you set requirements without versioning, pip will always install the latest version, but you can never be sure that the new version of a package is compatible with your code (or the code of any of the other requirements).
When you are confident that your code works as expected (all tests passes and the test-coverage is ~= 100%), you should run `pip freeze > requirements.txt` This will make sure that the requirements are frozen on the current versions, so the next guy installing this, or even you install it in the future in a new environment, won't have to worry about package-versions not being compatible.

**Comment by danielbentes** [comment]
Coming right up :)
